src/foam/u2/Element.js

```diff
@@ -2921,6 +2921,36 @@ foam.CLASS({
   ]
 });
 
+foam.CLASS({
+  package: 'foam.u2',
+  name: 'MenuViewRefinement',
```

Comment:
Why do we refine this rather than just include it in the original Menu model?

Reply:
Just following the pattern we use for Actions, no reason, assume it to split apart the view and the model